src/style/_form.scss

```diff
@@ -31,7 +31,7 @@
 
   &.validation-error {
     > label, > .title {
-      color: #9F3A38;
+      color: #CF0001;
```

Comment:
Would `#CF0000` be a better color? I don't think anyone will notice the blue `01` in the UI.